Makefile

```diff
@@ -50,8 +50,8 @@ vet: release-std
 	go vet $(pkgs)
 
 # will always run on some packages for a while.
-lintpkgs = ./modules ./modules/gateway ./modules/host ./modules/renter ./modules/renter/hostdb ./modules/renter/contractor \
-           ./node ./node/api/server ./persist ./siatest
+lintpkgs = ./modules ./modules/wallet ./modules/gateway ./modules/host ./modules/renter ./modules/renter/hostdb \
```

Comment:
generally speaking we try to sort long lists alphabetically, which means `modules/wallet` would come after `modules/renter/contractor`. Sorting just makes it easier to scan really long lists of things.

It looks like other parts of this list aren't 100% sorted either, if you want to fix that up at the same time.